lib/kubernetes-deploy/runner.rb

```diff
@@ -113,11 +116,15 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       end
 
       if verify_result
-        deploy_resources(resources, prune: prune, verify: true)
+        ::StatsD.measure('duration.normal_resources', tags: %W(namespace:#{@namespace} sha:#{@current_sha})) do
+          deploy_resources(resources, prune: prune, verify: true)
+        end
         record_statuses(resources)
         success = resources.all?(&:deploy_succeeded?)
       else
-        deploy_resources(resources, prune: prune, verify: false)
+        ::StatsD.measure('duration.normal_resources', tags: %W(namespace:#{@namespace} sha:#{@current_sha})) do
```

Comment:
I don't think we should measure in this case. If `verify_result` is false, we're skipping the polling step entirely, so it wouldn't be measuring any real deploy time (just the time of `kubectl apply` itself, basically).